cli/run/commandPlugins.ts

```diff
@@ -96,9 +99,10 @@ async function loadAndRegisterPlugin(
 			)}" for Rollup to recognize it.`
 		);
 	}
-	inputOptions.plugins!.push(
-		typeof plugin === 'function' ? plugin.call(plugin, pluginArg) : plugin
-	);
+	if (!Array.isArray(inputOptions.plugins)) {
```

Comment:
Again with the type changes, this should be unnecessary. Removing this check will then also fix coverage.